lab-1/lab-1.py

```diff
@@ -0,0 +1,14 @@
+def write_lines(task_text):
+    task_splited_text = task_text.split(',')
+
+    task_name = task_splited_text[0]
+    assignee_name = task_splited_text[1]
+    mentor_name = task_splited_text[2]
+    repeated_teacher_name = ','.join([mentor_name for i in range(45)])
```

Comment:
could use underscore in the comprehension as you don't need the value of **i**

[mentor_name for _ in range(45)]
```suggestion
    repeated_teacher_name = ','.join([mentor_name for _ in range(45)])
```